cms/topics/tests/test_blocks.py

```diff
@@ -215,9 +215,62 @@ def test_time_series_page_link_block_validation_fails_on_invalid_domain(self):
         with self.assertRaises(StructBlockValidationError) as info:
             block.clean(value)
 
-        patterns_str = " or ".join(settings.ONS_ALLOWED_LINK_DOMAINS)
+        allowed_domains = " or ".join(settings.ONS_ALLOWED_LINK_DOMAINS)
```

Comment:
This might be a pit nitpicky, but I think what would be better here is to do the following:
* Override the setting for `ONS_ALLOWED_LINK_DOMAINS` using a decorator
* Explicitly state the expected string on line 222

This way we also test that the code builds the error message correctly, as opposed to recreating the logic which builds the error message in the test (so "for a given value of `ONS_ALLOWED_LINK_DOMAINS`, this is the error message I expect"). Hope this makes sense!

(Also around line 258)